Assets/Fungus/Scripts/Editor/VariableConditionEditor.cs

```diff
@@ -33,61 +33,90 @@ public class VariableConditionEditor : CommandEditor
             new GUIContent(VariableUtil.GetCompareOperatorDescription(CompareOperator.NotEquals)),
         };
 
-        protected SerializedProperty compareOperatorProp;
-        protected SerializedProperty anyVarProp;
+        // protected SerializedProperty compareOperatorProp;
+        // protected SerializedProperty anyVarProp;
+        
+         protected SerializedProperty conditions;
 
         protected Dictionary<System.Type, SerializedProperty> propByVariableType;
 
         public override void OnEnable()
         {
             base.OnEnable();
 
-            compareOperatorProp = serializedObject.FindProperty("compareOperator");
-            anyVarProp = serializedObject.FindProperty("anyVar");
+            // compareOperatorProp = serializedObject.FindProperty("compareOperator");
+            // anyVarProp = serializedObject.FindProperty("anyVar");
+
+            conditions = serializedObject.FindProperty("conditions");
+            
+
         }
 
         public override void DrawCommandGUI()
         {
             serializedObject.Update();
 
-            VariableCondition t = target as VariableCondition;
+            EditorGUILayout.PropertyField(serializedObject.FindProperty("anyOrAllConditions"));            
 
-            var flowchart = (Flowchart)t.GetFlowchart();
-            if (flowchart == null)
-            {
-                return;
-            }
-
-            EditorGUILayout.PropertyField(anyVarProp, true);
+            conditions.arraySize = EditorGUILayout.IntField("Size", conditions.arraySize);
 
-            // Get selected variable
-            Variable selectedVariable = anyVarProp.FindPropertyRelative("variable").objectReferenceValue as Variable;
-            GUIContent[] operatorsList = emptyList;
-            if (selectedVariable != null)
-            {
-                operatorsList = selectedVariable.IsComparisonSupported() ? compareListAll : compareListEqualOnly;
-            }
-            
-            // Get previously selected operator
-            int selectedIndex = (int)t.CompareOperator;
-            if (selectedIndex < 0)
+            for (int i = 0; i < conditions.arraySize; i++)
             {
-                // Default to first index if the operator is not found in the available operators list
-                // This can occur when changing between variable types
-                selectedIndex = 0;
-            }
-
-            selectedIndex = EditorGUILayout.Popup(
-                new GUIContent("Compare", "The comparison operator to use when comparing values"),
-                selectedIndex,
-                operatorsList);
 
-            if (selectedVariable != null)
-            {
-                compareOperatorProp.enumValueIndex = selectedIndex;
+                GUILayout.Label("Condition "+(i+1).ToString(),EditorStyles.boldLabel);
```

Comment:
Some kind of heading before the condition list would be a good, but a heading per condition seems overkill. Ends up taking a lot of space.